docs/website/generator-errors.md

```diff
@@ -210,6 +212,8 @@ This usually indicates a bug in Xamarin.iOS/Xamarin.Mac; please [file a bug repo
 
 ### <a name='BI1116'/>BI1116: The parameter '*' in the delegate '*' does not have a [CCallback] or [BlockCallback] attribute. Defaulting to [CCallback]. Declare a custom delegate instead of using System.Action / System.Func and add the attribute on the corresponding parameter.
 
+### <a name='BI1117'/>BI1117: The * member is decorated with [Static] and its container class * is decorated with [Category] this leads to hard to use code. Please inline * into * class.
+
```

Comment:
missing doc ? or something not yet committed ?

Reply:
nope this was missing from de docs, I discovered it while adding the warning, I should make that clear in the commit message